
<launch>

  <node pkg="motor_imu_com" type="read_motor_imu" name="serial_moter_imu" output="screen">
  	<param name="Motor_dev" value="/dev/ttyUSB0" type = "str"/>
  	<param name="Imu_dev" value="/dev/ttyUSB0"  type = "str"/>
  	<param name="Motor_Speed" value="10" type = "int"/>
	<param name="Motor_baud" value="115200" type = "int"/>
	<param name="Imu_baud" value="1000000" type = "int"/>
  </node>

</launch>
